notebooks/1_table_oriented.ipynb

```diff
@@ -0,0 +1,484 @@
+{
```

Comment:
<p>A few comments:</p><p> </p><ul><li>I think would be nice to just use <code>df</code> as the name of this instead of <code>my_dataframe</code></li><li><code>I would defer introducing Categorical types until later</code></li><li><code>We don't need to specify the <code>index=</code> here as it matches the default</code></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Because <code>df</code> is such a widely used convention for naming DataFrames, I agree with @WillAyd that it would be better to use <code>df</code> than <code>my_dataframe</code>.</p><p> </p><p>I also agree that there's no need to specify the index here. I think that @TomAugspurger suggested specifying the index to avoid a RangeIndex, but the index is no longer being printed out in this notebook, so it's not an issue.</p>

Reply:
<ul><li><code>my_dataframe</code> -&gt; <code>df</code>; good point, adjusted</li><li>index is indeed of no more usage, removed</li><li>I added categorical to make sure that when later on applying the <code>describe</code> method it is not taken into account. To overcome this, I now used <code>Sex</code>as a column instead of <code>Pclass</code>. </li></ul>